ui/raidboss/data/05-shb/etc/the_diadem.js

```diff
@@ -20,6 +20,7 @@ export default {
       id: 'Diadem Found Gather Point',
       netRegex: NetRegexes.gameLog({ line: 'You sense a grade .* clouded (?:mineral deposit|rocky outcrop|mature tree|lush vegetation patch).*?', capture: false }),
       netRegexDe: NetRegexes.gameLog({ line: 'Du hast einen? verhüllten? (?:Nutzbaum|Vegetation|Steinbruch|Lagerstätte)? der Güte.*?', capture: false }),
+      netRegexFr: NetRegexes.gameLog({ line: 'Vous détectez une? (?:filon de minerai|carrière de pierre|arbre mature|carré de végétation) évanescente? de niveau 80 du côté (?:est|sud|ouest|nord|nord-est|sud-est|nord-ouest|sud-ouest).*?', capture: false }),
```

Comment:
```suggestion
      netRegexFr: NetRegexes.gameLog({ line: 'Vous détectez une? (?:filon de minerai|carrière de pierre|arbre mature|carré de végétation) évanescente?.*?', capture: false }),
```

Suggesting a shorter regex to match the en/de regexes.  What do you think?

Reply:
I'm not sure about my `regex` , I took @MaikoTan's advice and help. I had asked him if he should write the whole line. If you think your `regex` better, I would make the switch. Sorry, for `regex` I have to ask for help because I don't know anything about it. It is beyond my competence.

Reply:
It is ok.

I was just talking that I think it is safer to match up to "niveau 80 côte"(level 80 side) part.
But even this looks good to me!

Sorry for confusing.